packages/integration-tests/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sentry-browser-integration-tests",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "engines": {
+    "node": ">=10"
+  },
+  "private": true,
+  "scripts": {
+    "clean": "rimraf -g suites/**/dist",
+    "install-browsers": "playwright install --with-deps",
+    "lint": "run-s lint:prettier lint:eslint",
+    "lint:eslint": "eslint . --cache --cache-location '../../eslintcache/' --format stylish",
+    "lint:prettier": "prettier --check \"{suites,utils}/**/*.ts\"",
+    "test:ci": "playwright test ./suites --browser='all' --reporter='line'",
+    "pretest": "yarn clean",
+    "test": "playwright test ./suites"
+  },
+  "dependencies": {
+    "@playwright/test": "^1.17.0",
+    "@sentry/browser": "file:../browser",
```

Comment:
Do we still need this?

Reply:
Not currently, but will probably need in cases where we inject code to browser, at least not to break `eslint` and auto-completion in `test.ts`. Removed this, can add later along with said tests.